### PR TITLE
Hotfix/aut 2673/inline figure img to November release

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-2673

Backport of https://github.com/oat-sa/extension-tao-mediamanager/pull/453 and https://github.com/oat-sa/extension-tao-mediamanager/pull/457 to [November release](https://github.com/oat-sa/tao-community/blob/2022.11.4/composer.json#L57)

TO DO:

- [x] merge https://github.com/oat-sa/tao-item-runner-qti-fe/pull/343 and release
- [x] update css and js bundles

### CODE REVIEW

File: views/js/qtiCreator/renderers/Img.js [hunk 1]
```diff
@@ -19,16 +19,22 @@
...
+    'taoQtiItem/qtiCreator/helper/findParentElement'
+], function (_, Renderer, Widget, FigureWidget, FigureModel, findParentElement) {
     'use strict';
 
     const CreatorImg = _.clone(Renderer);
 
     CreatorImg.render = function render(img, options) {
         const $container = Renderer.getContainer(img);
-        if ($container.parent('figure').length) {
+        if ($container.parent('figure').length || $container.parent('span').length && $container.parent('span').data('figure')) {
```
Comment (Silvia-loza) [inline views/js/qtiCreator/renderers/Img.js:33]: Discrepancy with original PR is this expected?

**Review by Silvia-loza — COMMENTED** [review]
missing bundles

File: views/js/qtiCreator/renderers/Img.js (hunk 1 above)
Comment (bziondik) [reply views/js/qtiCreator/renderers/Img.js:33]: Here backport for 2 PRs, this changes from second https://github.com/oat-sa/extension-tao-mediamanager/pull/457

**Review by Karol-Stelmaczonek — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**